vega-embed 6.20.4 breaks downstream package build

In JupyterLab, the application of the patch is breaking our CI. This is most certainly due to the usage of a multi packages repository.

```sh
Error: Patch file found for package yallist which is not present at node_modules/yallist
---
patch-package finished with 1 error(s).

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vega-embed@6.20.4 postinstall: `npx patch-package`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the vega-embed@6.20.4 postinstall script.
```

One comment on the logic, a better pattern would be to pin to a working older `yallist` version, to publish your own version of `yallist` or to wait for a patch upstream. The current approach of patching a dependency on the flight sounds pretty dangerous. Moreover this could black list you in some organization because you apply changes outside your scope.

--- a/package.json
+++ b/package.json
@@ -78,6 +78,9 @@
     "vega-themes": "^2.10.0",
     "vega-tooltip": "^0.27.0"
   },
+  "bundledDependencies": [
+    "yallist"
+  ],
   "scripts": {
     "prebuild": "yarn clean && yarn build:style",
     "build": "rollup -c",
@@ -90,12 +93,11 @@
     "pretest": "yarn build:style",
     "test": "beemo jest --stdio stream",
     "test:inspect": "node --inspect-brk ./node_modules/.bin/jest --runInBand",
-    "prepare": "beemo create-config",
+    "prepare": "beemo create-config && npx patch-package",
     "prettierbase": "beemo prettier '*.{css,scss,html}'",
     "eslintbase": "beemo eslint .",
     "format": "yarn eslintbase --fix && yarn prettierbase --write",
     "lint": "yarn eslintbase && yarn prettierbase --check",
-    "release": "yarn build && auto shipit",
-    "postinstall": "npx patch-package"
+    "release": "yarn build && auto shipit"
   }
 }

